FlightsFile.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System.Text;
+
+namespace AirportTicketBooking.CSVFiles
+{
+    public static class FlightsFile
+    {
+        private const string FlightsFilePath = "Flights.csv";
+
+        public static bool HasData
+        {
+            get
+            {
+                var fileInfo = new FileInfo(FlightsFilePath);
+                return fileInfo.Exists && fileInfo.Length > 0;  
+            }
+        }
+
+        public static void Append(Flight newFlight)
+        {
+            if (Exists(newFlight.ID))
+                throw new Exception("This flight already exists.");
+
+            File.AppendAllText(FlightsFilePath, $"{newFlight.ID}, " +
+                $"{newFlight.AirplaneCapacity}, {newFlight.Price}, " +
+                $"{newFlight.DepartureCountry}, {newFlight.DepartureDate.Month}-" +
+                $"{newFlight.DepartureDate.Day}-{newFlight.DepartureDate.Year}, " +
+                $"{newFlight.DepartureAirport}, {newFlight.DestinationCountry}, " +
+                $"{newFlight.ArrivalAirport}, {newFlight.Class}\n");
+        }
+
+        public static void ImportCSVFile(string newFlightsFilePath)
+        {
+            try
+            {
+                RemoveAll();
+                IsValid(newFlightsFilePath);
+            }
+            catch (Exception ex) 
+            {
+                throw ex;
+            }
+        }
+
+        public static bool IsValid(string newFilePath)
```

Comment:
this method performs 2 things: validating and importing date from the file
we need to separate here
method for importing, and method (in other class - Flights Validator) for validating the imported row to ensure it is valid or not